ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -15,9 +15,6 @@ import StatusQ.Core.Utils 0.1 as SQUtils
 QtObject {
     id: root
 
-    // Required until tokensStore, walletAssetsStore and currentActivityFiltersStore are reworked
-    required property SharedStores.NetworksStore networksStore
```

Comment:
This was a problem because we had broken token lists. This property was never set.